### DISCUSSION TITLE
alpine.init create a couple of alpine.data, how

### DISCUSSION THREAD

**Discussion by ericxin1982** [alpinejs/alpine#3627] [1. Help] [answered]
Hi All,

I saw alpine.init can generate alpine.data, but all example just create one alpine.data object, if I need to register a couple of alpine.data during alpine.init, not sure how to do, someone can teach me ?

thanks
Eric Xin

**Comment by ekwoka** [comment]
Just do it multiple times.

It's just javascript.

If you want to run a function multiple times, then run it multiple times.

**Reply by ericxin1982** [reply]
Can you give me sample code, multiple time means multiple init + multiple alpine.data, or one init + multiple alpine.data?

**Reply by ekwoka** [reply, marked as answer]
The second

```js
document.addEventListener('alpine:init', () => {
   Alpine.data('foo', () => ({
     foo: 'bar'
   })

  Alpine.data('fizz', () => ({
    fizz: 'buzz'
  })
})
```
